lib/grpc_mock/mocked_call.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'grpc'
+
+module GrpcMock
+  class MockedCall
+    attr_reader :deadline, :metadata
+
+    def initialize(metadata: {})
```

Comment:
Yeah, at first I tried to reproduce every method `ActiveCall` exposes and it turned out that is a waste of time. Keeping deadline in the initializer since someone might find it useful. d26c345